server/sonar-server/src/main/java/org/sonar/server/ce/ws/SubmitAction.java

```diff
@@ -79,13 +89,16 @@ public void define(WebService.NewController controller) {
 
   @Override
   public void handle(Request wsRequest, Response wsResponse) throws Exception {
+    String organizationKey = wsRequest.getParam(PARAM_ORGANIZATION_KEY)
+      .emptyAsNull()
+      .or(() -> defaultOrganizationProvider.get().getKey());
```

Comment:
![MINOR](https://raw.githubusercontent.com/SonarCommunity/sonar-github/master/images/severity-minor.png) Replace this lambda with a method reference. [![rule](https://raw.githubusercontent.com/SonarCommunity/sonar-github/master/images/rule.png)](https://next.sonarqube.com/sonarqube/coding_rules#rule_key=squid%3AS1612)